alphashield/trading/orchestrator.py

```
@@ -1,120 +1,133 @@
+            "loan_id": loan_id,
+            "coverage_ratio": float(cr),
+            "template": template_name,
+            "weights": {k: float(v) for k, v in target_weights.to_dict().items()},
+            "costs": float(execution.get("total_cost", 0.0)),
+            "metrics": risk_metrics,
+        }
+        try:
+            self.logger.info(json.dumps(log))
+            decisions_inc(template_name)
+        except Exception:
+            pass
```

Ruff (0.14.1)

`try`-`except`-`pass` detected, consider logging the exception

(S110)